src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/client/addressindex/model/AddressIndexAddressSplitDTO.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.ons.ctp.integration.contactcentresvc.client.addressindex.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AddressIndexAddressSplitDTO {
```

Comment:
Please could we have some javadoc here to explain the purpose of this class.
Am wondering what 'split' is about

Reply:
There is already a class AddressIndexAddressDTO, and associated AddressIndexResponseDTO, AddressIndexSearchResultsDTO. These are used to unmarshal the present AIMs returned JSON to an object representation. I created the three classes AddressIndexAddressSplitDTO, AddressIndexResponseSplitDTO and AddressIndexSearchResultsSplitDTO. They were named with "Split" introduced into the class names as the main purpose of the new AIMs endpoint called was to take the formatted address originally returned as one line and "split" these into census address fields addressLine1, addressLine2, addressLine3, postcode and region.  Happy to rename, or add Javadoc stating contains address split into census address frame fields. Though I believe the logic used by AIMS might not be identical.

Reply:
I came here to add the same comment. I found the "split" naming hard to fathom.

Reply:
Added some javadoc. Name follows convention of previous classes with split added. Happy to rename if think something else more suitable.

Reply:
The more I think about it the more I think we should find a more suitable name for this Split class.
CCSvc will be concerning itself with PUT /cases whose address can be SPLIT you see, and I think it muddy the water.

AddressIndexAddressComposite?
AddressIndexAddressSuperset?
AddressIndexAddressComplete?
AddressIndexAddressVerbose?

Take your pick - but not Split please.

Reply:
Done, AddressIndexAddressComposite it is then.